### ISSUE TITLE
Add Run button to Tutorials

### ISSUE THREAD

**Issue by jeremycook** [solidjs/solid-site#112]
The outcome of changing code in some tutorials happen before I realize it, and I miss it. The workaround I've found is to click on the Output tab and then click back to the Result tab. This reruns Results without losing any code changes. Having a Run button would be more intuitive and help in these situations.

This tutorial is an example of when it helps to rerun after making code changes to see what is going on: https://www.solidjs.com/tutorial/async_suspense_list

**Comment by EmNudge** [comment]
To add onto this, I've noticed some bugs in the past that were resolved by refreshing the page. I later realized I could switch tabs for the same result.

Working with the Audio API and `setInterval` also necessitates constantly refreshing. If we're unable to cancel the interval and close the audio stream, it will persist through to future changes in the REPL and create an unpleasant experience.

Another avenue might be to add an option to refresh the entire REPL on code changes, throwing away any audio streams or intervals with it.
